common.mk

```diff
@@ -12,6 +12,9 @@ LD_FLAGS += $(LDFLAGS)
 ifeq (,$(findstring nostrip,$(COMETBFT_BUILD_OPTIONS)))
   BUILD_FLAGS += -trimpath
   LD_FLAGS += -s -w
+else
+  #prepare for delve
+  BUILD_FLAGS+= -gcflags "all=-N -l"
```

Comment:
I'm not sure I get why this is an `else` and not an independed "if" with its own option.
The "else" here means we want to strip the binary... but usually, we want to do the opposite for debugging

Reply:
Actually, we strip in the if, because the comparison is with an empty string, so in the else we do not strip and add the extra flags.
But given that this is the only case in which we compare with the empty string and that now we have an else, I will invert the two cases. Thanks for raising it.